lib/protos.js

```diff
@@ -334,8 +333,16 @@ exports._wrapInitialize = function() {
  */
 
 exports._wrapPage = function() {
-  // Noop the first page call if integration assumes pageview
-  if (this._assumesPageview) return this.page = after(2, this.page);
+  var page = this.page;
+  this.page = function() {
+    if (this._assumesPageview) {
+      if (this._initialPageSkipped) return page.apply(this, arguments);
+
+      return this._initialPageSkipped = true;
+    }
+
+    return page.apply(this, arguments);
```

Comment:
Is there a need to store `initialPageSkipped` on the instance? It could be simplified to:
```js
  var page = this.page;
  var initialPageSkipped = false;
  this.page = function() {
    if (this._assumesPageview && !initialPageSkipped) {
      return initialPageSkipped = true;
    }

    return page.apply(this, arguments);
```

Reply:
good point. fixed!